assets/Product.wxs

```diff
@@ -31,6 +31,13 @@
 
     <!-- Set properties for add/remove programs -->
     <Property Id="ARPHELPLINK" Value="$(var.InfoURL)" />
+
+	<!-- Checkbox to allow starting PowerShell after the installation (in UI mode only) -->
+	<Property Id="WIXUI_EXITDIALOGOPTIONALCHECKBOXTEXT" Value="Open PowerShell" />
```

Comment:
Good idea. Done and manually tested on the MSI artifact of CI to verify the CheckBox text is still 'Open PowerShell' after using a variable for it.